src/ScriptCs.Core/Repl.cs

```diff
@@ -192,12 +192,52 @@ private void HandlePaketReferences(IEnumerable<string> references)
         {
             var paketRefs = references.Where(r => r.StartsWith(Constants.PaketPrefix)).Select(r => r.Substring(Constants.PaketPrefix.Length)).ToArray();
             var refs = Path.Combine(FileSystem.CurrentDirectory, "Refs.csx");
-            var fi = new FileInfo(refs);
-            dynamic result = PaketShim.ResolveLoadScript(fi, paketRefs, s =>
+            
+            // todo: is this the .csx we are currently evaluating?
+            var scriptFileBeingProcessed = new FileInfo(refs);
```

Comment:
Outside the REPL we do have a script name, and we can use that for the hash, though that code will appear elsewhere.